samples/databus/custom-serializer/Core_6/Receiver/MessageWithLargePayloadHandler.cs

```diff
@@ -0,0 +1,15 @@
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+
+public class MessageWithLargePayloadHandler :
+    IHandleMessages<MessageWithLargePayload>
+{
+    static ILog log = LogManager.GetLogger<MessageWithLargePayloadHandler>();
+
+    public Task Handle(MessageWithLargePayload message, IMessageHandlerContext context)
+    {
+        log.Info($"Message received, size of blob property: {message.LargeData.Value.Length} measurements");
```

Comment:
Copied from the `FileShareDatabus` sample. I'll reword this one to make it `N measurements received` or something.